CODE_OF_CONDUCT.md

```diff
@@ -1,101 +1,104 @@
+6. Stay on topic: Please take heated argument with little chance of resolution (such as politics or emacs vs vim!) to a thread in #⁠the-forum.
+7. Don't be a dick.
 
- * Violence, threats of violence or violent language directed against another person.
- * Sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory jokes and language.
- * Posting or displaying sexually explicit or violent material.
- * Posting or threatening to post other people's personally identifying information ("doxing").
- * Personal insults, particularly those related to gender, sexual orientation, race, religion, or disability.
- * Inappropriate photography or recording.
- * Inappropriate physical contact. You should have someone's consent before touching them.
- * Unwelcome sexual attention. This includes, sexualized comments or jokes; inappropriate touching, groping, and unwelcomed sexual advances.
- * Deliberate intimidation, stalking or following (online or in person).
- * Advocating for, or encouraging, any of the above behavior.
- * Sustained disruption of community events, including talks and presentations.
+## Unwanted Opinions and Ideologies
+The following ideologies or opinions are not welcome.
 
-## 5. Weapons Policy
+- Fascism
+- Colonialism
+- White supremacy
+- Religious extremism
+- Nationalism
+- Racism
+- Homophobia
+- Transphobia
 
-No weapons will be allowed at No Boilerplate events, community spaces, or in other spaces covered by the scope of this Code of Conduct. Weapons include but are not limited to guns, explosives (including fireworks), and large knives such as those used for hunting or display, as well as any other item used for the purpose of causing injury or harm to others. Anyone seen in possession of one of these items will be asked to leave immediately, and will only be allowed to return without the weapon. Community members are further expected to comply with all state and local laws on this matter.
+Talking about them from a outside perspective is allowed, for example discussing how to deal with Homophobic relatives.
 
-## 6. Consequences of Unacceptable Behavior
+### Safe Space
 
-Unacceptable behavior from any community member, including sponsors and those with decision-making authority, will not be tolerated.
+This community is a safe space for the following:
+- LGBTQIA+
+- neurodivergent (ADHD, Autism, etc.)
 
-Anyone asked to stop unacceptable behavior is expected to comply immediately.
+##### Examples of behavior that contributes to creating a positive environment include:
 
-If a community member engages in unacceptable behavior, the community organizers may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community without warning (and without refund in the case of a paid event).
+-   Being kind and courteous to others
+-   Using welcoming and inclusive language
+-   Being respectful of differing viewpoints and experiences
```

Comment:
I think this here might create a paradox of intolerance. POSSIBLY needs clarification, but probably not.

Reply:
:+1:

Reply:
i'm going to shorten it to just "be respectful" - there really isnt any need to clarify about what